tests/validation/Tests.java

```diff
@@ -0,0 +1,97 @@
+package validation;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class Tests {
+
+    PasswordChecker passwordChecker;
+    PhoneValidator phoneValidator;
+    EmailValidator emailValidator;
+
+    @BeforeEach
+    void setUp() {
+        passwordChecker = new PasswordChecker();
+        phoneValidator = new PhoneValidator();
+        emailValidator = new EmailValidator();
+    }
+
+    @Test
+    void TestIfPasswordLengthIsAtLeastX() {
```

Comment:
General comment that applies to all tests below. It is good to separate negative and positive tests into separate unit tests, so that when they fail you know exactly what went wrong. Here on the surface it is a bit of guesswork whether it was positive or negative that failed.